tests/sles4sap/trento/test_hana_unregister_node.pm

```diff
@@ -0,0 +1,86 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Trento unregister the secondary node
+# Maintainer: QE-SAP <qe-sap@suse.de>, Michele Pagot <michele.pagot@suse.com>
+
+use strict;
+use warnings;
+use Mojo::Base 'publiccloud::basetest';
+use base 'consoletest';
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use qesapdeployment;
+use trento;
+
+sub run {
+    my ($self) = @_;
+    select_serial_terminal;
+
+    my $secondary_host = 'vmhana02';
+    cluster_print_cluster_status($secondary_host);
+
+    # Set secondary node in maintenance mode
+    my $prov = get_required_var('PUBLIC_CLOUD_PROVIDER');
+    qesap_ansible_cmd(cmd => "sudo crm configure property maintenance-mode=true",
+        provider => $prov,
+        filter => $secondary_host);
+    cluster_wait_status_by_regex($secondary_host, 900, m/global.+true/, "true");
```

Comment:
Now that the function receives 1 regexp instead of 2, I guess this call will need to be changed to:
```
cluster_wait_status_by_regex($secondary_host, 900, qr/global.+true/);
```
Regarding `qr//` vs. `m//`, sorry I did not comment on this before. Using `m//` will make Perl try to match the regexp at the moment of the call to whatever is in `$_` and then pass the result of that operation to the function. This is not what you want ... what you want to do is pass the regexp to the function. This can be accomplished with `qr//` (quote regexp).

See more details here: https://perldoc.perl.org/perlop#Regexp-Quote-Like-Operators

Reply:
thx for the comment, refactored.